distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -270,6 +270,15 @@ public String toString() {
 
     public static final DynamicPort RMI_REG_PORT = new DynamicPort(5);
 
+    public static final AbstractIntegrationTest.DynamicPort AMQP_PORT =
```

Comment:
Doesn't choosing the 101th port mean that every test will try and allocate and hold 11 blocks of (10 each) ports, so blocking out 110 ports for each test? Should we be trying to use lower port number?